client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -240,6 +240,11 @@ export default function getThankYouPageUrl( {
 		return redirectUrlForPostCheckoutUpsell;
 	}
 
+	if ( urlFromCookie?.startsWith( '/start/setup-site' ) ) {
```

Comment:
All the return values of this function are covered by unit tests. Could we add a test for this in https://github.com/Automattic/wp-calypso/blob/aae5e45748fc4302494e8f89e5d595cd3635e4b4/client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js#L41